lib/actions/keys.ts

```diff
@@ -48,7 +48,7 @@ Examples:
 `,
 	permission: 'user',
 	async action(params) {
-		const key = await getBalenaSdk().models.key.get(params.id);
+		const key = await getBalenaSdk().models.key.get(parseInt(params.id, 10));
```

Comment:
Instead of doing these two `parseInt` calls, can't we just change the capitano arg type for this command?
From:
```
export const info: CommandDefinition<{ id: string }> = {
```
To:
```
export const info: CommandDefinition<{ id: number }> = {
```

Reply:
Yes, good point.

Reply:
Just tested that, and it seems to allow though strings(!?)  E.g., with this change:
```
$balena key 15561r
BalenaRequestError: Request error: Database error
```
Not really sure what's going on there with capitano, but I suggest we stick with the parseInt for now, since this will be converted to oclif shortly anyway.

Reply:
Imo `$balena key 15561r` should error since that's not a correct key id.
What if you have a key named as `1234mysshkey` and also have a different ssh key with the db id `1234`. If the user runs `$balena key 1234mysshkey` the `parseInt` will parse the 1234 and delete the key that happens to have that db id.

Reply:
Ok let me add some better checks there in a separate PR.